tests/tap.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python3
```

Comment:
I wish Meson would support TAP so that we didn't have to copy this script to every repo... but the issue is open, nothing to do but either wait or implement it ourselves